charts/icm-as/templates/as-deployment.yaml

```diff
@@ -0,0 +1,213 @@
+          {{- end -}}
+        {{- end -}}
+      {{- end -}}
+      {{- if or (.Values.image.secret) ($customizationImagePullSecrets) }}
+      imagePullSecrets:
+        - name: {{ .Values.image.secret }}
+        {{- range $curr := $customizationImagePullSecrets }}
+        - name: {{ $curr }}
+        {{- end }}
+      {{- end }}
+      containers:
+        - name: icm-as-server
+          image: "{{ .Values.image.repository }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          command:
+          - "/bin/sh"
+          - "-c"
+          - {{- toYaml .Values.image.command | nindent 12 }}
+          env:
+          - name: IS_DBPREPARE
+            value: "false"
+          - name: SERVER_NAME
+            value: "appserver"
+          {{- if .Values.debug.enabled }}
+          - name: DEBUG_ICM
+            value: "true"
+          {{- end }}
+          {{- range $key, $value := .Values.environment }}
+          - name: {{ $key }}
+            value: {{ $value | quote }}
+          {{- end }}
+          ports:
+            # Servlet engine connector port
+            - name: http
+              containerPort: 7743
+              protocol: TCP
+          {{- if .Values.debug.enabled }}
+            # Java Debug port
+            - name: debug
+              containerPort: 7744
+              protocol: TCP
+          {{- end }}
+            # JMX port
+            - name: jmx
+              containerPort: 7745
+              protocol: TCP
+          resources:
+            {{- toYaml .Values.resources.appServer | trim | nindent 12 }}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: /intershop/license/license.xml
+              name: license-volume
+              readOnly: true
+              {{- if eq .Values.license.type "configMap" }}
+              subPath: license.xml
+              {{- else if eq .Values.license.type "csi" }}
+              subPath: license
+              {{- end }}
+            - mountPath: /intershop/sites
+              name: sites-volume
+          {{- if .Values.customdata.enabled }}
+            - mountPath: {{ .Values.customdata.mountPoint }}
+              name: custom-data-volume
+          {{- end }}
+            - mountPath: /intershop/customizations
+              name: customizations-volume
+            - mountPath: /intershop/jgroups-share
+              name: jgroups-volume
+          {{- if hasKey .Values.environment "STAGING_SYSTEM_TYPE" }}
+          {{- if eq .Values.environment.STAGING_SYSTEM_TYPE "editing" }}
+            - mountPath: /intershop/replication-conf/replication-clusters.xml
+              name: replication-volume
+              readOnly: true
+              subPath: replication-clusters.xml
+          {{- end }}
+          {{- end }}
+          startupProbe:
+            httpGet:
+              path: /status/LivenessProbe
```

Comment:
I guess that's on purpose? The startup probe using the same http resource as the liveness probe?

Reply:
I changed it to "/", which looks better for a startup probe